src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -38,12 +38,17 @@ const Breadcrumbs = ({ query = '', type, bibUrl, itemUrl, edd }) => {
       return crumbs;
     }
 
-    crumbs.push(
-      <li key="search">
-        <Link to={`${baseUrl}/search?${query}`} onClick={() => onClick('Search Results')}>
-          Search Results
-        </Link>
-      </li>);
+    const stringifiedQuery = query.replace(/^q=/, "");
+
+    if (stringifiedQuery && stringifiedQuery !== "undefined") {
```

Comment:
The `stringifiedQuery` string itself becomes "undefined"? Or do you mean to do `typeof stringifiedQuery !== "undefined"`?

Reply:
We were running into "searchKeywords=undefined" in the url while developing SHEP.

Reply:
How does that happen?...

Reply:
Just realized I had one more change to the code to make in `ResultsList` which highlights how this was happening. That change might be enough to keep this from happening again.